src/mame/jaleco/tetrisp2.cpp

```diff
@@ -74,33 +74,27 @@ To Do:
 
 ***************************************************************************/
 
-u16 tetrisp2_state::rockn_adpcmbank_r()
+u16 rockn_state::rockn_adpcmbank_r()
 {
 	return ((m_rockn_adpcmbank & 0xf0ff) | (m_rockn_protectdata << 8));
 }
 
-void tetrisp2_state::rockn_adpcmbank_w(u16 data)
+void rockn_state::rockn_adpcmbank_w(u16 data)
 {
-	u8 *SNDROM = memregion("ymz")->base();
-	int bank;
-
 	m_rockn_adpcmbank = data;
-	bank = ((data & 0x001f) >> 2);
+	int bank = ((data & 0x001f) >> 2);
 
 	if (bank > 7)
 	{
 		popmessage("!!!!! ADPCM BANK OVER:%01X (%04X) !!!!!", bank, data);
 		bank = 0;
 	}
```

Comment:
It’s completely impossible for `bank` to be greater than 7 here – `0x1f >> 2 == 0x07`.  The test is useless.